spec/lib/cyclonedx/report_python_modules_cyclonedx_spec.rb

```diff
@@ -0,0 +1,255 @@
+require_relative '../../spec_helper'
+require 'json'
+
+describe Cyclonedx::ReportPythonModules do
+  describe "#run" do
+    it 'should report all the deps in the unpinned requirements.txt' do
+      repo = Salus::Repo.new('spec/fixtures/python/requirements_unpinned')
+      scanner = Salus::Scanners::ReportPythonModules.new(repository: repo, config: {})
+      scanner.run
+
+      python_cyclonedx = Cyclonedx::ReportPythonModules.new(scanner.report)
+      expected = [
+        {
+          "bom-ref": "pkg:python_requirement/requests",
+          "type": "library",
+          "group": "",
+          "name": "requests",
+          "version": ">=2.5",
+          "purl": "pkg:python_requirement/requests",
+          "properties": [
+            {
+              "key": "source",
+              "value": ""
+            },
+            {
+              "key": "dependency_file",
+              "value": "requirements.txt"
+            }
+          ]
+        },
+        {
+          "bom-ref": "pkg:python_requirement/six",
+            "type": "library",
+            "group": "",
+            "name": "six",
+            "version": ">=1.9",
+            "purl": "pkg:python_requirement/six",
+            "properties": [
+              {
+                "key": "source",
+                "value": ""
+              },
+              {
+                "key": "dependency_file",
+                "value": "requirements.txt"
+              }
+            ]
+        },
+        {
+          "bom-ref": "pkg:python_requirement/pycryptodome",
+            "type": "library",
+            "group": "",
+            "name": "pycryptodome",
+            "version": ">=3.4.11",
+            "purl": "pkg:python_requirement/pycryptodome",
+            "properties": [
+              {
+                "key": "source",
+                "value": ""
+              },
+              {
+                "key": "dependency_file",
+                "value": "requirements.txt"
+              }
+            ]
+        }
+      ]
+      expect(python_cyclonedx.build_components_object).to include(*expected)
+    end
+
+    it 'should report all the deps in the pinned requirements.txt' do
+      repo = Salus::Repo.new('spec/fixtures/python/requirements_pinned')
+      scanner = Salus::Scanners::ReportPythonModules.new(repository: repo, config: {})
+      scanner.run
+
+      python_cyclonedx = Cyclonedx::ReportPythonModules.new(scanner.report)
+      expected = [
+        {
+          "bom-ref": "pkg:python_requirement/amqp",
+          "type": "library",
+          "group": "",
+          "name": "amqp",
+          "version": "==2.2.2",
```

Comment:
Maybe remove `==` so that we have this?
```json
"version": "2.2.2",
```
What do you think, Josh?

Reply:
Yep I can remove `==` for pinned requirements. It's just for unpinned/semi-pinned requirements, if something like 

`>=3.2.1` 

is given for example, getting rid of `>=` might cause confusion as we wouldn't be able to tell that any newer version (3.2.1+) is also specified?